time                     stamp                 # Timestamp relative to the trajectory initial state.
geometry_msgs/Pose       pose                  # Pose of the body frame expressed in the world frame T_WB in [m].
geometry_msgs/Twist      twist                 # Velocity in the body frame v_B in [m/s].
geometry_msgs/Vector3    linear_acceleration   # Linear acceleration in the world frame a_W in [m/s^2].
